.github/workflows/reusable_checks_protobuf.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           pixi-version: v0.39.0
 
       - name: Fetch latest main (so we can grab the current schema snapshot)
-        run: git fetch origin main --depth=1
+        run: time git fetch origin main # yes, we need full --depth for `buf` to work
 
       # TODO(cmc): I would prefer some kind of continue-on-error behavior here, but every
       # "native" solution I've tried sucked, and I really don't want to introduce yet another
```

Comment:
> every "native" solution I've tried sucked

Can you elaborate? Or at least describe what you'd want to see here

Reply:
try `if: success() || failure()` instead of `continue-on-error`. it should mark the whole job as failed but still run those steps

Reply:
:eyes: thanks I'll try that

Reply:
perfect :man_cook::kiss: thanks @jprochazk